### ISSUE TITLE
Gallery Block sorting not preserved in "Edit Gallery" modal until reload

### ISSUE THREAD

**Issue by SamReciter** [WordPress/gutenberg#14477]
**Describe the bug**
If you sort your gallery pictures via drag&drop in "Edit Gallery" modal and then again open "Edit Gallery" modal, the sorting is reseted (or isn't changed beforehand). Same if you "Publish"|"Update" the page, but don't reload/refresh the website. Only after "Publish"|"Update" the page and then reload/refresh the website, the sorting is preserved to the "Edit Gallery" modal.

**To Reproduce**
1. Create page.
2. Add Gallery Block, select pictures and click "Create new gallery".
3. Open the "Edit Gallery" modal - change sorting of the pictures via drag&drop - click "Update gallery".
4. Reopen the "Edit Gallery" modal directly or after "Publish"|"Update" - the sorting is not preserved.
5. If you reopen the "Edit Gallery" modal after "Publish"|"Update" and then refreshing the website - the sorting is preserved correctly.

**Expected behavior**
Same behaviour as after "Publish"|"Update" and then refreshing the website aka the sorting is preserved when opening "Edit Gallery" modal again.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Google Chrome Version 72.0.3626.121 (Offizieller Build) (64-Bit)

**Additional context**
- WordPress 5.0.4 & WordPress 5.1.1

**Comment by danielpickett** [comment]
I can confirm that I am able to reproduce the issue on WordPress 5.1.1 without Gutenberg. I have also now tested this issue using a fresh build of the Gutenberg plugin from the master branch of this repo, as well as using the current version of the plugin from the wordpress.org plugins repository. I am not able to reproduce the issue using the latest versions of the Gutenberg plugin. The issue seems to have been resolved.
